src/settings/MatchProfiles/MatchProfiles.js

```diff
@@ -378,18 +378,18 @@ export class MatchProfiles extends Component {
       const ordersModuleVersion = getModuleVersion(records, 'Orders Business Logic Module');
       const notesModuleVersion = getModuleVersion(records, 'Notes');
       const invoiceModuleVersion = getModuleVersion(records, 'Invoice business logic module');
-      const SRMModuleVersion = getModuleVersion(records, 'Source Record Manager Module');
+      // const SRMModuleVersion = getModuleVersion(records, 'Source Record Manager Module');
```

Comment:
should we remove this three commented lines?

Reply:
done